src/cpp/py_bindings/CMakeLists.txt

```diff
@@ -11,17 +21,17 @@ execute_process(
 find_package(nanobind CONFIG REQUIRED)
 
 
-file(GLOB BINDINGS_SOURCES ./*.cpp)
-file(GLOB BINDINGS_HEADERS ./*.hpp)
+file(GLOB BINDINGS_SOURCES src/vision_api/*.cpp)
+file(GLOB BINDINGS_HEADERS src/vision_api/*.hpp)
 
-nanobind_add_module(py_model_api NB_STATIC STABLE_ABI LTO ${BINDINGS_SOURCES} ${BINDINGS_HEADERS})
+message(INFO ${BINDINGS_SOURCES})
 
-target_link_libraries(py_model_api PRIVATE model_api)
+nanobind_add_module(_vision_api NB_STATIC STABLE_ABI LTO ${BINDINGS_SOURCES} ${BINDINGS_HEADERS})
 
-nanobind_add_stub(
-    py_model_api_stub
-    MODULE py_model_api
-    OUTPUT py_model_api.pyi
-    PYTHON_PATH $<TARGET_FILE_DIR:py_model_api>
-    DEPENDS py_model_api
+set_target_properties(_vision_api PROPERTIES
+    LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/src/vision_api"
```

Comment:
Binaries are dumped to the source dir, which looks a bit weird. Can we use the build dir for that?